OXRS-BMD-SmokeDetector-ESP32-FW.ino

```diff
@@ -457,9 +367,9 @@ void publishEvent(uint8_t index, char * type, char * event)
   uint8_t channel = getChannel(index);
 
   // Show event on screen
-  char display[32];
-  sprintf_P(display, PSTR("idx:%2d %s %s   "), index, type, event);
-  screen.show_event(display);
+//  char display[32];

```

Comment:
FYI - I have moved this inside the `Rack32` library on my local machine (where I have moved `Rack32` into a stand alone library). The `Rack32` library checks if the JSON payload has `index`, `type` and `event` fields and if so generates an event for displaying on the LCD.